website/content/en/docs/building-operators/ansible/tutorial.md

```diff
@@ -146,201 +94,143 @@ Modify `roles/memcached/tasks/main.yml` to look like the following:
-
-### Build and run the operator
-
-Before running the operator, Kubernetes needs to know about the new custom
-resource definition the operator will be watching.
+This memcached role will:
+- Ensure a memcached Deployment exists
+- Set the Deployment size
 
-Deploy the CRD:
+It is good practice to set default values for variables used in Ansible
+Roles, so edit `memcached-operator/roles/memecached/defaults/main.yml`:
 
-```sh
-$ make install
+```yaml
+---
+# defaults file for Memcached
+size: 1
 ```
 
-Once this is done, there are two ways to run the operator:
+Finally, update the `Memcached` sample, `memcached-operator/config/samples/cache_v1alpha1_memcached.yaml`:
 
-- As a pod inside a Kubernetes cluster
-- As a go program outside the cluster
-
-#### 1. Run as a pod inside a Kubernetes cluster
-
-Running as a pod inside a Kubernetes cluster is preferred for production use.
-
-Build the memcached-operator image and push it to a registry:
-```
-$ make docker-build docker-push IMG=quay.io/example/memcached-operator:v0.0.1
+```yaml
+apiVersion: cache.example.com/v1alpha1
+kind: Memcached
+metadata:
+  name: memcached-sample
+spec:
+  size: 3
 ```
 
-Kubernetes deployment manifests are generated in `config/manager/manager.yaml`.
+The key-value pairs in the Custom Resource spec are passed
+to Ansible as extra variables.
 
-Deploy the memcached-operator:
+__Note:__ The names of all variables in the spec field are converted to
+snake_case by the operator before running ansible. For example,
+serviceAccount in the spec becomes service_account in ansible. You can
+disable this case conversion by setting the `snakeCaseParameters` option
+to `false` in your `watches.yaml`. It is recommended that you perform some
+type validation in Ansible on the variables to ensure that your
+application is receiving expected input.
 
-```sh
-$ make deploy IMG=quay.io/example/memcached-operator:v0.0.1
-```
+#### Finishing up
 
-Verify that the memcached-operator is up and running:
+All that remains is building and pushing the operator container to your favorite registry.
 
-```sh
-$ kubectl get deployment -n memcached-operator-system
-NAME                                        DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-memcached-operator-controller-manager       1         1         1            1           1m
+``` sh
+$ IMG=quay.io/someuser/image:v1.0.0 make docker-build && make docker-push
 ```
 
-#### 2. Run outside the cluster
-
-This method is preferred during the development cycle to speed up deployment and testing.
-
-**Note**: Ensure that [Ansible Runner][ansible-runner-tool] and [Ansible Runner
-HTTP Plugin][ansible-runner-http-plugin] is installed or else you will see
-unexpected errors from Ansible Runner when a Custom Resource is created.
-
-Run the operator locally with the default Kubernetes config file present at
-`$HOME/.kube/config`:
+**Note** To allow the cluster to pull the image, the repository needs to be set to public. 
```

Comment:
You could also configure the cluster to be able to pull images from a private repo, we don't need to explain how to do that but might be good to mention that they don't **have** to use publicly accessible images

Reply:
```suggestion
**Note** Make sure the repository is accessible by the cluster.
```

Reply:
I other docs, I added "or you must configure an image pull secret."